### PR TITLE
Add toggle password visibility

### LINKED ISSUE #369: [Accessibility] : Ability to toggle Password Visibility
As an EmbeddedChat developer

**I need to**:
I think we  should add the eye icon to toggle the encryption of password (which is very common in a lot of logging in interfaces.)
currently it looks like this 
image

**So That**:
Adding the eye icon, will be helpful if anyone mistypes the password or want to double check rather than typing the password again from zero
**Acceptance Criteria**

- [x] Add the eye icon (make sure its responsive)
- [x] Add the toggling logic to it

### PR DESCRIPTION
**Pull request by Akshun-01**
# Feature to toggle between Password Visibility
I added the eye icon (closed and open) to toggle between password visibility and added logic to ensure proper functioning!

## Acceptance Criteria fulfillment

- [x] Add the eye icon (make sure its responsive)
- [x] Add the toggling logic to it

Fixes #369 

## Video/Screenshots
image

image

### CODE REVIEW

File: packages/react/src/components/auth/LoginForm.js [hunk 1]
```diff
@@ -87,7 +97,49 @@ export default function LoginForm() {
...
+                type={showPassword ? 'text' : 'password'}
+                onChange={handleEditPassword}
+              />
+              <Box
+                type="button"
+                css={passwordEyeCss}
+                onClick={handleTogglePassword}
+              >
+                {showPassword ? (
+                  <svg
```
Comment (sidmohanty11) [inline packages/react/src/components/auth/LoginForm.js:110]: Lets move the svgs to a different file

**Review by sidmohanty11 — COMMENTED** [review]
Thanks @Akshun-01 ! Looks awesome, left a minor comment

File: packages/react/src/components/auth/LoginForm.js (hunk 1 above)
Comment (Akshun-01) [reply packages/react/src/components/auth/LoginForm.js:110]: hi, do we have any common assets folder or some other rules for adding svg's ?

**Comment by Akshun-01** [comment]
Also @sidmohanty11 , could please check the comment on the main issue, I added some other things that can be added to login. Should I open different issues for each, or continue adding here only (new feature with each commit) ?

File: packages/react/src/components/auth/LoginForm.js (hunk 1 above)
Comment (Akshun-01) [reply packages/react/src/components/auth/LoginForm.js:110]: @sidmohanty11, can you please clarify so that I can move forward with this issue.
Thanks!

**Comment by Akshun-01** [comment]
Hello @sidmohanty11 , looked around and found how other svg's are exported. Did the same with these.
Could you please review the changes! ^^

**Review by sidmohanty11 — APPROVED** [review]
Thanks @Akshun-01 !

**Pull request merged**